src/node/desktop/src/ui/widgets/choose-r.ts

```diff
@@ -81,8 +82,8 @@ export class ChooseRModalWindow extends ModalDialog<CallbackData | null> {
     const r32 = findDefault32Bit();
     const r64 = findDefault64Bit();
     const initData = {
-      default32bitPath: isValidInstallation(r32) ? r32 : '',
-      default64bitPath: isValidInstallation(r64) ? r64 : '',
+      default32bitPath: isValidInstallation(r32, 'i386') ? r32 : '',
```

Comment:
There are changes missing to this file. See https://github.com/rstudio/rstudio/pull/12970